src/test/resources/mocks/magistrates_public_list/magistratesPublicList.json

```diff
@@ -26,10 +26,6 @@
     {
       "courtHouse": {
         "courtHouseName": "This is the site name",
-        "courtHouseContact": {
-          "venueTelephone": "01234 123 123",
-          "venueEmail": "a@b.com"
-        },
```

Comment:
There's a couple of mags lists in the functional test data folder that will also need updating